drivers/gles2/rasterizer_canvas_base_gles2.cpp

```diff
@@ -947,11 +947,15 @@ void RasterizerCanvasBaseGLES2::draw_lens_distortion_rect(const Rect2 &p_rect, f
 
 void RasterizerCanvasBaseGLES2::initialize() {
 
-	bool flag_stream = false;
-	if (flag_stream)
-		_buffer_upload_usage_flag = GL_STREAM_DRAW;
-	else
-		_buffer_upload_usage_flag = GL_DYNAMIC_DRAW;
+	int flag_stream_mode = GLOBAL_GET("rendering/2d/opengl/legacy_stream");
+	switch (flag_stream_mode) {
+		default: {
+			_buffer_upload_usage_flag = GL_DYNAMIC_DRAW;
+		} break;
```

Comment:
Style wise I think it's better if you have separate cases for `default` and `1` as you've done in other files. Makes the intent clearer.